src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -36,6 +36,8 @@ class OutcomeCardContainer extends Component {
     super(props);
     const {outcome: {content}} = props;
     this.state = {
+      isEditingText: false,
+      isEditingMeta: false,
```

Comment:
Yeah this is the part that I spent the most time thinking over. The problem is that child components can't assume that they own the editing state; they can only signal back to this container component that _they_ are done editing. That way if you have a modal open when the editor loses focus (this case), they don't race to turn off the whole card's edit state.

A more general solution would be for the child components to call a generic `signalEditing(isEditing, uid)` API and for the container to keep track of children which may be editing the card. That way you could keep an atomic piece of state (a set) to manage all the clients. I'd be happy to write that up too. I generally try to wait until there are > 2 use cases for a pattern before applying it, but it could be fun to do anyways :)